docs/DeveloperGuide.md

```diff
@@ -1069,22 +1108,18 @@ This command will exit UniCa$h.
 
 ---
 
-[//]: # (## 8. Individual Contributions and Efforts)
-
-[//]: # ()
-[//]: # (Here's what each of us have contributed. &#40;Might not need this section&#41;)
-
-[//]: # (---)
-
 ## Appendix
 
-### Proposed Changes
-
-To be added
-
-### Known JavaFX Bugs
+### Planned Enhancements
```

Comment:
We might have to discuss and edit this part later on, we are limited to 10 planned enhancements (2 x 5 members) total and enhancements cannot be new features, and must meet these criteria, just a point to note. Most important being a feature that exists must be tweaked in order for it to qualify.

![image](https://github.com/AY2324S1-CS2103-T16-3/tp/assets/19762596/d76842a1-de2a-4be4-a109-dcf1c0b6b35a)


 We can have a separate section on planned future features.
![image](https://github.com/AY2324S1-CS2103-T16-3/tp/assets/19762596/a9aa9417-2943-4a05-b11c-dc24f6cede4c)

Will link this comment to the discussions page

Reply:
Sure

Reply:
<img width="970" alt="Screenshot 2023-11-13 at 14 23 54" src="https://github.com/AY2324S1-CS2103-T16-3/tp/assets/29497029/47129672-3dd3-4579-845a-9bd62c29f081">

Moved some of the planned enhancements under potential features